tutorial/Lab3/README.md

```diff
@@ -0,0 +1,82 @@
+# Lab 3. Keeping track of the deployed application
+
+Lets say you have deployed different release versions of your application (i.e. you upgraded the running application). How do you keep track of the versions and how can you do a rollback?
+
+# Revision management using Kubernetes directly
```

Comment:
Is the `directly` necessary? It's not enough to just say `Revision management with Kubernetes`?

Reply:
Good point. Removed.